### PR TITLE
Add the JMeta project

### PR DESCRIPTION
**Pull request by nmcodeeu**
Call for contributions on the JMeta project.
Thank you up-for-grabs.net

### CODE REVIEW

File: _data/projects/jmeta [hunk 1]
```diff
@@ -0,0 +1,12 @@
+name: JMeta
+desc: JMeta is a free software project, written in Java, aiming to provide a search platform and a framework for building fully-decentralized P2P search applications.
+site: https://gitlab.ouahpiti.info/meta/jmeta
```
Comment (shiftkey) [inline _data/projects/jmeta]: There's an issue with the HTTPS configuration on this server, so you might have problems with visitors accessing the site...

**Comment by shiftkey** [comment]
The file needs to be renamed to end in `.yml` so the parser can load it correctly...

**Comment by nmcodeeu** [comment]
Thank you. I just renamed  the file.
I am aware of the HTTPS issue. We are using a self-signed certificate so there is a security warning from browsers for now. We will fix this soon.

File: _data/projects/jmeta.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+name: JMeta
+desc: JMeta is a free software project, written in Java, aiming to provide a search platform and a framework for building fully-decentralized P2P search applications.
+site: https://gitlab.ouahpiti.info/meta/jmeta
+tags:
+- p2p
+- "search engine"
+- DHT
+- metadata
+- up-for-grabs
```
Comment (shiftkey) [inline _data/projects/jmeta.yml]: don't worry about this tag

File: _data/projects/jmeta.yml [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+upforgrabs:
+  name: JMeta issues
```
Comment (shiftkey) [inline _data/projects/jmeta.yml:13]: Is there a more specific label here for the tags you're using on the project?

Chrome and Safari are actively blocking me from accessing the site, so I'm not quite sure how you've organised things...

Comment (nmcodeeu) [inline _data/projects/jmeta.yml:13]: Thank you for the feedback. We will have to fix this quickly as our issues management takes place there.
I've put our github page instead.

File: _data/projects/jmeta.yml [hunk 4]
```diff
@@ -1,12 +1,14 @@
...
+- "data modeling"
+- "collaborative seach engine"
+- Tomp2p
 - DHT
-- metadata
-- up-for-grabs
+- Java
+- p2p
 upforgrabs:
   name: JMeta issues
```
Comment (shiftkey) [inline _data/projects/jmeta.yml:13]: If you don't have a preferred tag here, I'd recommend using `up-for-grabs`, which means the `link` below will become `https://github.com/dyslesiq/jmeta/labels/up-for-grabs`. You'd need to create this on the project site as well...

**Comment by shiftkey** [comment]
Closing this out as inactive. Feel free to open a fresh PR if you'd still like to see your project listed.

**Pull request closed without merging**